src/tcgwars/logic/impl/gen8/BattleStyles.groovy

```diff
@@ -2123,8 +2132,8 @@ public enum BattleStyles implements LogicCardInfo {
           energyCost FIGHTING, FIGHTING, COLORLESS
           attackRequirement {}
           onAttack {
-            discardAllSelfEnergy(null)
             multiDamage(opp.all, 2, 120)
+            discardAllSelfEnergy(null)
```

Comment:
This has to be in an afterDamage closure. Everything not in an afterDamage closure happens before damage.

Reply:
Good point, I'll wrap it in an afterDamage closure

Reply:
Tegull mentioned some other cards that need this treatment too.